frontend/app/routes/_gcweb-app.tsx

```diff
@@ -343,6 +295,26 @@ function Breadcrumbs() {
   );
 }
 
+function MenuOption({ onClick, icon, href, i18nKey }: { onClick: MouseEventHandler<HTMLAnchorElement>; icon?: ReactElement; href: string; i18nKey: string }) {
+  return (
+    <Link className="font-body hover:text-blue-hover ring-blue-hover flex h-[55px] items-center rounded-sm px-4 ring-offset-2 focus:border-none focus:outline-none focus:ring-2" onClick={onClick} to={href} aria-label={i18nKey}>
+      {icon}
+      {i18nKey}
```

Comment:
This isn't being used as an i18n *key* here, it's a pre-translated string. As such, I would just name this `text` or something like that.